test/test.js

```diff
@@ -2904,6 +2904,65 @@ describe('Page', function() {
   });
 });
 
+
+describe('ElementHandle', function() {
+  let browser;
+  let page;
+
+  beforeAll(SX(async function() {
```

Comment:
Just put `describe('ElementHandle.$', ..)` and `describe('ElementHandle.$$', ..)` test suites somewhere next to other ElementHandle test suites. This will save you work for setting up a browser, and also will make tests faster since you'll avoid re-launching the browser.